tests/phpunit/tests/formatting/removeAccents.php

```diff
@@ -109,4 +109,20 @@ public function test_transcribe_serbian_crossed_d() {
 		$this->assertSame( 'DJdj', remove_accents( 'Đđ', 'sr_RS' ) );
 		$this->assertSame( 'Dd', remove_accents( 'Đđ' ) );
 	}
+
+	/**
+	 * @ticket 24661
+	 *
+	 * Tests Unicode sequence normalization from NFD (Normalization Form Decomposed)
+	 *   to NFC (Normalization Form [Pre]Composed), the encoding used in `remove_accents()`.
+	 *
+	 * For more information on Unicode normalization, see
+	 *   https://unicode.org/faq/normalization.html.
+	 */
+	public function test_remove_accents_latin1_supplement_nfd_encoding() {
+		$input  = 'ªºÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ';
+		$output = 'aoAAAAAAAECEEEEIIIIDNOOOOOOUUUUYTHsaaaaaaaeceeeeiiiidnoooooouuuuythy';
+
+		$this->assertSame( $output, remove_accents( $input ), 'remove_accents replaces Latin-1 Supplement with NFD encoding' );
```

Comment:
Looks good! Just a niptick :) 
If the functionality depends on `function_exists()` (`intl` support in PHP), perhaps the test should too? Can probably be marked as skipped if no support.

Reply:
Good tip 👍🏻 Thanks!

I've dropped in a [`@requires`](https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#skipping-tests-using-requires) to make sure `intl` is supported (c78aedb), else it will skip the test.